test/apps/iodemo/run_io_demo.sh

```diff
@@ -299,12 +299,33 @@ build_server_args_list() {
 			iodemo_server_args+=" $key $value"
 			shift
 			;;
+		-k)
+			value="$2"
+			iodemo_server_args+=" $key $value"
+			shift
+			;;
+		-r)
+			value="$2"
+			iodemo_server_args+=" $key $value"
+			shift
+			;;
+		-b)
+			value="$2"
+			iodemo_server_args+=" $key $value"
+			shift
+			;;
```

Comment:
now there are many params
i think we can unite args like this:
```
-d|-P|-k| ... )
			value="$2"
			iodemo_server_args+=" $key $value"
			shift
			;;
-q|-v|...)
			iodemo_server_args+=" $key"
			;;
```

Reply:
fixed